@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -323,26 +312,33 @@ export namespace DisplacementFeature {
     }
 }
 
-export class DisplacementFeatureMixin implements DisplacementFeature {
+export class DisplacementFeatureMixin implements DisplacementFeature, MixinShaderProperties {
     needsUpdate?: boolean;
     uniformsNeedUpdate?: boolean;
     defines?: any;
     shaderDefines?: any;
     shaderUniforms?: UniformsType;
     onBeforeCompile?: CompileCallback;
-    private m_displacementMap?: THREE.Texture;
+    private m_displacementMap: THREE.Texture | null = null;
 
-    protected getDisplacementMap(): THREE.Texture | undefined {
+    get displacementMap(): THREE.Texture | null {
```

Comment:
What about:
```
DisplacementFeatureMixin.addDisplacementProperties()
```
which already defines (adds) this getter/setter?

Reply:
I needed to add it there so that tslint didn't complained about a missing property from the DisplacementFeature interface. 
I've added a comment to clarify that.